tests/test_utility/test_file_utils/test.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2019 Intel Corporation
```

Comment:
Can we add these unit tests in the respective module directory by creating folder unit_tests that module? If future if move these module to some other directory then we can module as it is.

Reply:
That's what I started with. But later found out that there was a tests directory at the root, so decided to rather put it along side the other tests. I will raise another PR to refactor a few thing, might do it there. There is a lot of scope for improvement. For example we could make the tests modular enough to be run with choice or all at once. Needs some thorough thinking.